### PR TITLE
Userguide sync doc/readme

### PR DESCRIPTION
**Pull request by Ayelet41**

### CODE REVIEW

**Comment by simonz130** [comment]
@Ayelet41 you need to rebase this PR so it includes only what you intended to include and not changes in 85 files.
Also please update the description to describe what this PR is supposed to include, so we know what to review.

**Comment by Ayelet41** [comment]
Reverted the irrelevant changes and the *html files.

**Comment by daniel-sanche** [comment]
Can you add a description for what you intended to change here? It looks like this PR might have some unintended changes in it (it overlaps with existing PRs), but it's hard to tell without a description

**Comment by Ayelet41** [comment]
@daniel-sanche The PRs are complementary and this dependent on the previous, I separated to 2 PRs as they were to address different requirements, but I can combine if needed.

In this PR I aligned the content from the old user guide (docs/Readme.md), the Tutorial and the new User Guide( website).
This included changes that were made between the versions e.g. "SLIs, SLOs and Burn rate Alerts" documentation that was added after the creation of the website.
I also updated links that refer to the old docs/readme file.

**Comment by Ayelet41** [comment]
The PRs are complementary and this should follow the naming alignment.  
The intention is to align the content from old docs/Readme file to the new website -User guide so the old one can be deprecated. 
For this purpose I align content, added the SLI, SLOs and SRE Recipes sections that were missing and direct the links to the new user guide (in the website).

To clarify I didn't introduce new content only copied the content from docs/readme and the tutorial(Neo).

File: website/content/en/docs/getting_started.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ weight: 2
 
 Click the Cloud Shell button for automated one-click installation of a new Sandbox cluster in a new Google Cloud Project.
 
-[![Open in Cloud Shell](http://www.gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git&cloudshell_git_branch=v0.5.0&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image/uncertified:v0.5.0&cloudshell_tutorial=docs/tutorial.md)
+[![Open in Cloud Shell](http://www.gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git&cloudshell_git_branch=v0.5.0&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image/uncertified:v0.6.0&cloudshell_tutorial=docs/tutorial.md)
```
Comment (daniel-sanche) [inline website/content/en/docs/getting_started.md:28]: Why does the git branch use v0.5.0 when the image uses v0.6.0?

Comment (Ayelet41) [reply website/content/en/docs/getting_started.md]: By mistake - Reverted

File: website/content/en/docs/User Guide/Learn Ops Management/cloud_trace.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ weight: 40
 
 #### Trace Overview
 
-Cloud Trace ([documentation](https://cloud.google.com/trace/docs/)) enables developers to see distributed traces that visually expose latency bottleneck in requests. Developers instrument application code to collect trace information. You can also include environmental information in traces and trace information can be included in Cloud Logging logs. The Trace UI can then pull relevant log events into the trace timelines. 
+[Cloud Trace](https://console.cloud.google.com/traces)([documentation](https://cloud.google.com/trace/docs/)) enables developers to see distributed traces that visually expose latency bottleneck in requests. Developers instrument application code to collect trace information. You can also include environmental information in traces and trace information can be included in Cloud Logging logs. The Trace UI can then pull relevant log events into the trace timelines.
```
[the remark this answers is not in the document]
Comment (Ayelet41) [reply website/content/en/docs/User Guide/Learn Ops Management/cloud_trace.md]: Added space.

**Comment by Ayelet41** [comment]
@simonz130 @daniel-sanche  I rebased with master as there were a lot of changes I also created a new branch with only needed changes userguid_to_docs_sync, let me know if you prefer me to closed this and open a new PR for new branch.

**Pull request closed without merging**